spec/fake_stripe/requests/stripe_js_spec.rb

```diff
@@ -31,7 +31,7 @@
 
       response = Net::HTTP.get(url)
 
-      expect(response).to include "class Element"
+      expect(response).to include "class StripeElement"
```

Comment:
Are there going to be any implications to our trees app test suite around this change?

Reply:
Not that I can find. The trees app doesn't refer to this class directly, and all tree tests pass with this new gem